packages/smartcontracts/src/tests/BridgeProxy.flushFund.spec.ts

```diff
@@ -46,6 +48,41 @@ describe('Test flushFund functionalities', () => {
     expect(balanceETHAfterFlush.sub(balanceETHBeforeFlush)).to.equal(toWei('40'));
   });
 
+  it('Should flush the funds successfully when there is initial redundant funds per `tokenAddress`', async () => {
```

Comment:
```suggestion
  it('Should flush the funds per token successfully when there is initial redundant funds per `tokenAddress`', async () => {
```

This is for the per token scenario?

Reply:
Oh yes, fixed